modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -118,7 +118,17 @@ v = f_y \times y'' + c_y
 tangential distortion coefficients. \f$s_1\f$, \f$s_2\f$, \f$s_3\f$, and \f$s_4\f$, are the thin prism distortion
 coefficients. Higher-order coefficients are not considered in OpenCV.
 
-The next figures show two common types of radial distortion: barrel distortion (typically \f$ k_1 < 0 \f$) and pincushion distortion (typically \f$ k_1 > 0 \f$).
+The next figures show two common types of radial distortion: barrel distortion
+(1 + k_1 r^2 + k_2 r^4 + k_3 r^6 monotonically decreasing)
+and pincushion distortion (1 + k_1 r^2 + k_2 r^4 + k_3 r^6 monotonically increasing).
+ Radial distortion is always monotonic for real lenses,
+ and if the estimator produces a non monotonic result,
+ this should be considered a calibration failure.
+ More generally, radial distortion must be monotonic and the distortion function, must be bijective.
+ A failed estimation result may look deceptively good near the image center
+ but will work poorly in e.g. AR/SFM applications.
+The optimization does not include these constraints as the framework does not support
```

Comment:
```suggestion
The optimization method used for camera calibration does not include these constraints as the framework does not support
```

Reply:
@midjji
> Regarding the suggestion, there isn't a opt framework in opencv which
supports it, so technically correct as is. Though it's unimportant. Shall I
fix it?

What do you mean? Do you want to change the `framework` word in the second part of the sentence?
If you feel that the suggestion is not relevant, feel free to ignore it.

At the end, it is just a minor suggestion.